metrics/web/lib/common/presentation/metrics_theme/model/light_metrics_theme_data.dart

```diff
@@ -113,6 +114,22 @@ class LightMetricsThemeData extends MetricsThemeData {
             successfulColor: ColorConfig.primaryColor,
             failedColor: ColorConfig.accentColor,
           ),
+          barGraphPopupThemeData: const BarGraphPopupThemeData(
+            color: Colors.white,
+            shadowColor: _shadowColor,
+            titleTextStyle: MetricsTextStyle(
```

Comment:
It seems like title and subtitle styles are the same for both light and dark themes - would it be better to move them to the `TextStyleConfig`?